cmd/terramate-lsp/main.go

```diff
@@ -69,3 +92,22 @@ type readWriter struct {
 }
 
 func (s *readWriter) Close() error { return nil }
+
+func configureLogging(logLevel string, logFmt string, output io.Writer) {
+	zloglevel, err := zerolog.ParseLevel(logLevel)
+
+	if err != nil {
+		zloglevel = zerolog.FatalLevel
```

Comment:
hmm wont defaulting to fatal level lead to surprising behavior ? like I have a typo like "debg" as the level and I never see any logs and get puzzled thinking something else is wrong :thinking: